File: src/backend/VoteBem/Entities/RedeSocial.cs
namespace VoteBem.Entities
{
    public class RedeSocial
    {
        public long SqCandidato { get; private set; }
        public int NrOrdem { get; private set; }
        public string? DsUrl { get; private set; }
        public string? TipoRedeSocial { get; private set; }

        public Candidatura Candidatura { get; private set; } = null!;

        protected RedeSocial() { }
    }
}

File: src/etl/models/rede_social.py
from sqlalchemy import Column, BigInteger, String, Integer, ForeignKey, PrimaryKeyConstraint
from .base import Base

class RedeSocial(Base):
    __tablename__ = "rede_social"

    sq_candidato = Column(BigInteger, ForeignKey("candidatura.sq_candidato"), nullable=False)
    nr_ordem = Column(Integer, nullable=False)
    ds_url = Column(String)
    tipo_rede_social = Column(String)

    __table_args__ = (
        PrimaryKeyConstraint("sq_candidato", "nr_ordem"),
    )
